packages/@react-spectrum/table/src/Resizer.tsx

```diff
@@ -1,30 +1,52 @@
 /* eslint-disable jsx-a11y/role-supports-aria-props */
 import {classNames} from '@react-spectrum/utils';
-import {FocusRing} from '@react-aria/focus';
-import React from 'react';
+import {GridNode} from '@react-types/grid';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
+import React, {RefObject} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/table/vars.css';
+import {useLocale, useMessageFormatter} from '@react-aria/i18n';
 import {useTableColumnResize} from '@react-aria/table';
 import {useTableContext} from './TableView';
 
+interface ResizerProps<T> {
+  column: GridNode<T>,
+  tableRef: RefObject<HTMLElement>,
+  showResizer: boolean,
+  onResizeDone: () => void,
+  onResizeEntered: () => void
+}
+
+function Resizer<T>(props: ResizerProps<T>, ref: RefObject<HTMLDivElement>) {
+  let {column, tableRef, showResizer} = props;
+  let {state, columnState} = useTableContext();
+  let formatMessage = useMessageFormatter(intlMessages);
+  let {direction} = useLocale();
+
+  let {resizerProps} = useTableColumnResize({...props, label: formatMessage('columnResizer')}, {...state, ...columnState}, ref);
 
-function Resizer(props, ref) {
-  const {column} = props;
-  let {columnState} = useTableContext();
-  let {resizerProps} = useTableColumnResize({column}, columnState, ref);
+  let style = {
+    cursor: undefined,
+    height: '100%',
+    display: showResizer ? 'block' : 'none'
+  };
+  // always be 100% height? never? only while dragging? only while focused?
```

Comment:
IMO, only when dragging/focused but I'm not too bothered having it 100% on hover